chat-with-rendezvous/chat.go

```diff
@@ -5,22 +5,21 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"log"
 	"os"
+	"sync"
 
-	logging "github.com/ipfs/go-log"
 	"github.com/libp2p/go-libp2p"
 	"github.com/libp2p/go-libp2p-discovery"
 	libp2pdht "github.com/libp2p/go-libp2p-kad-dht"
 	inet "github.com/libp2p/go-libp2p-net"
 	"github.com/libp2p/go-libp2p-peerstore"
 	"github.com/libp2p/go-libp2p-protocol"
-	"github.com/multiformats/go-multiaddr"
+	multiaddr "github.com/multiformats/go-multiaddr"
 )
 
-var log = logging.Logger("rendezvous")
```

Comment:
please keep this log around, we use this for libp2p logging and we'd like the logs to be altogether.